Pargi/MapViewController.swift

```diff
@@ -13,6 +13,8 @@ import Pulley
 class MapViewController: UIViewController, MKMapViewDelegate, UIGestureRecognizerDelegate, PulleyPrimaryContentControllerDelegate {
     @IBOutlet var mapView: MKMapView!
     @IBOutlet var mapPanGesture: UIPanGestureRecognizer!
+    @IBOutlet weak var locateUserButton: UIButton!
```

Comment:
The `weak` here is accurate, which I like, however, could you please also change the two other vars to match the correct paradigm (sort of out of scope, but might as well fix it).

Reply:
Sure. Will add the weak for the others as well. There doesn’t seem to be a strong consensus on whether to use weak or not for IBOutlet’s.